libs/blocks/tabs/tabs.css

```diff
@@ -264,6 +263,61 @@
   height: 24px;
 }
 
+.tabs .tab-paddles {
+  position: relative;
+  z-index: 3;
+}
+
+.tabs .tab-paddles .paddle {
+  position: absolute;
+  width: 32px;
+  height: 48px;
+  bottom: 0;
+  background: #fff;
```

Comment:
Looking good. Few things i'm seeing. 
1) We should consider the dark theme. both `.dark .tabs` and `.tabs.dark` and style the overflow arrows accordingly. 
2) There should be a :hover state for these including a cursor: pointer, and some type of icon color hover indication. Probably just follow the tab button text hover colors.

Reply:
There is no official dark theme for tabs implemented yet but I used the existing dark tabs colors.